pkg/controller/ttl/ttl_controller.go

```diff
@@ -195,7 +195,7 @@ func (ttlc *Controller) deleteNode(obj interface{}) {
 func (ttlc *Controller) enqueueNode(node *v1.Node) {
 	key, err := controller.KeyFunc(node)
 	if err != nil {
-		klog.Errorf("Couldn't get key for object %+v", node)
+		klog.ErrorS(nil, "Couldn't get key for object", "node", node)
```

Comment:
If we're logging the entire object, we shouldn't use `ErrorS`: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#use-klogkobj-and-klogkref-for-kubernetes-objects

Reply:
I'm going to use what @navidshaikh commented.